### DISCUSSION TITLE
Reprogramming of F-Buttons?

### DISCUSSION THREAD

**Discussion by Engesser** [freeplane/freeplane#681] [Help] [answered]
F-Buttons facilitate life. But how can I reprogramme the F-Buttons, i.e. give them a new function, adding new menu command? With an unused F-Button no problem, but what if it is already used with a menu item?

Thanks for help

Karl

**Comment by euu2021** [comment, marked as answer]
To assign hotkeys, use `Tools->Assign hot key`:
![javaw_sSMKB84nXJ](https://user-images.githubusercontent.com/77707706/192069506-4de62fe9-6a21-4368-9c48-edb958ef8807.gif)

Or, simply use `Ctrl + click` in the desired menu item, or result in the Command Search

**Reply by Engesser** [reply]
Thanks!
I always used the Programs F-button and not the keyboard ones!! Now it works, misunderstandings are everywhere!

Greetings

Karl
